server/the_commander.py

```diff
@@ -0,0 +1,173 @@
+import re
+import shlex
+import subprocess
+
+from sqlalchemy import desc
+
+import king_phisher.plugins as plugin_opts
+import king_phisher.server.database.models as db_models
+import king_phisher.server.plugins as plugins
+import king_phisher.server.signals as signals
+
+
+class Plugin(plugins.ServerPlugin):
+    authors = ['Corey Gilks']
+    title = 'The Commander'
+    description = """
+    Execute an action from the KP Server after new credentials are received. Originally this plugin was created to
+    quickly authenticate to the targets VPN after new credentials are received. When the target is using
+    MFA every second counts, so action must be taken quickly. Operators may be unable to respond fast enough therefore
+    this plugin is needed.
+    
+    You can dynamically include the username, password and MFA values in your command by using the following python 
+    format string syntax:
+
+    {0} = Username
```

Comment:
If you use kwargs on L158, you can use the names as the variables. That would make the command a lot easier to read.